.gitignore

```diff
@@ -3,6 +3,8 @@
 **.json
 **.pwbi
 archive/
+tests/
```

Comment:
There isn't actually a tests/ directory yet so this isn't doing anything. But as we progress we will likely want to keep all test files in a common directory anyways. 

Keep.